selftests/job_api/test_features.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python3
+
+import os
+
+from avocado import Test
+from avocado.core import exit_codes
+from avocado.core.job import Job
+from avocado.core.suite import TestSuite
+
+BOOLEAN_ENABLED = [True, 'true', 'on', 1]
+BOOLEAN_DISABLED = [False, 'false', 'off', 0]
+
+
+class JobAPIFeaturesTest(Test):
+
+    def check_directory_exists(self, path):
+        """Check if a directory exists"""
+        self.assertTrue(os.path.isdir(path))
```

Comment:
What about add to all assert functions the error message for a better understanding of the problem?

Reply:
@willianrampazzo can you keep track of @richtja suggestion as a later improvement a patch?

Reply:
Sure!